webview-ui/package.json

```diff
@@ -69,7 +68,7 @@
 		"storybook": "^8.5.2",
 		"ts-jest": "^27.1.5",
 		"typescript": "^4.9.5",
-		"vite": "^5.4.14"
+		"vite": "^6.0.5"
```

Comment:
did this broke hrm ?

Reply:
HMR wasn't working within VSCode in Vite v6, but was in Vite v5. I figured out that the issue was CORS-related, and I was able to fix it, so we're back to v6 (thankfully).